tasks/broad/ImputationBeagleTasks.wdl

```diff
@@ -1,5 +1,47 @@
 version 1.0
 
+task InferContigsFromVcf {
```

Comment:
might want to rename as this is doing more than just infering contigs from the vcf, its also doing the overlap with `allowed

Reply:
good idea, maybe `InferValidContigsFromVcf` ?